aesthetic.css

```diff
@@ -57,25 +57,27 @@ nav {
   align-items: center;
 }
 
-nav > * {
-  margin-right: 0.6rem;
-}
-
-nav > a:last-child {
-  margin-right: 0;
-}
-
 a.logo {
-  color :black;
-  font-size: 4rem;
   user-select: none;
-  line-height: 1;
   border-bottom: none;
-  padding-bottom: 0;
+  width: 74px;
+  height: 74px;
 }
 
 a.logo:hover {
-  color: #f2c311;
+  filter: invert(74%) sepia(74%) saturate(511%) hue-rotate(356deg) brightness(93%) contrast(104%);
```

Comment:
This filter (courtesy of https://codepen.io/sosuke/pen/Pjoqqp) recreates the link hover effect that previously worked fine when the logo was a character. Now that it is an SVG, this filter effect ensures the same 'turn yellow' effect occurs when the logo is moused over.